spark_pipeline_framework/transformers/fhir_sender/v1/fhir_sender_processor.py

```diff
@@ -66,9 +67,60 @@ def send_partition_to_server(
         request_id_list: List[str] = []
         responses: List[Dict[str, Any]] = []
         if FhirSenderOperation.operation_equals(
+            operation, FhirSenderOperation.FHIR_OPERATION_PATCH
+        ):
+            for item in json_data_list:
+                item_value = json.loads(item["value"])
+                payload = [
+                    json.loads(convert_dict_to_fhir_json(payload_item))
```

Comment:
do we need to run convert_dict_to_fhir_json on payload_item? Seem to be a redundant step stirungying and json.loading again.